docs/api.md

```diff
@@ -1293,14 +1292,6 @@ to use the API after this request has succeeded.
 
 ### Emails
 
-#### GET /recovery_email/check_can_add_secondary_address
```

Comment:
Seems like my precommit hooks have broken, this should have been removed in https://github.com/mozilla/fxa-auth-server/pull/2234